Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -757,6 +757,35 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .AddParameter("object", _("Object"), "Drawer")
       .AddParameter("expression", _("X drawing position"))
       .AddParameter("expression", _("Y drawing position"));
-
+  obj.AddAction("SetAntialiasing",
```

Comment:
Keep a new line between function calls to be consistent with the rest of the file.
```suggestion

  obj.AddAction("SetAntialiasing",
```